src/main/scala/org/clulab/reach/display/package.scala

```diff
@@ -1,12 +1,14 @@
 package org.clulab.reach
 
+import com.typesafe.scalalogging.LazyLogging
 import org.clulab.odin._
 import org.clulab.reach.mentions._
 import org.clulab.processors.{Document, Sentence}
+
 import scala.util.matching.Regex
 
 
-package object display {
+package object display extends LazyLogging {
```

Comment:
Sorry about not being more specific when we discussed this issue, but the `display` package should be using `println`, not `logger.info`.  Can you please revert your changes to this file?